### ISSUE TITLE
Extra wrapping in op:start-session.acceptable-location-sig

### ISSUE THREAD

**Issue by dpwiz** [ocapn/ocapn-test-suite#3]
The spec says:

> The `acceptable-location-sig` is the signature of the serialized `acceptable-location`.

However, [the code](https://github.com/ocapn/ocapn-test-suite/blob/f4892052d5379583cb925e8ba26cb3df73f4abc8/utils/captp.py#L44) wraps the acceptable-location in an extra record: `<my-location [...]>`.

**Comment by tsyesika** [comment]
Yeah, you're right. I think I initially left it off because I believed it was unnecessary and should rather be changed in goblins, however I spoke to Christine a while ago about it and she explained why this is needed. Basically that if you sign something without providing the context in which you're signing it, it could potentially be used in other contexts by someone else.

I'll update the spec to align with how this actually works in both Goblins and the test suite.
